js/ftx.js

```diff
@@ -1925,6 +1928,39 @@ module.exports = class ftx extends Exchange {
         return result;
     }
 
+    async cancelOrders (ids, symbol = undefined, params = {}) {
+        /**
+         * @method
+         * @name ftx#cancelOrders
+         * @description cancel multiple orders
+         * @param {[string]} ids order ids
+         * @param {string|undefined} symbol not used by ftx cancelOrders ()
+         * @param {object} params extra parameters specific to the ftx api endpoint
+         * @returns {object} raw - a list of order ids queued for cancelation
+         */
+        await this.loadMarkets ();
+        // https://docs.ccxt.com/en/latest/manual.html#user-defined-clientorderid
+        const clientOrderIds = this.safeValue2 (params, 'client_order_ids', 'clientOrderIds');
+        params = this.omit (params, [ 'client_order_ids', 'clientOrderIds' ]);
+        const request = {};
+        if (ids !== undefined) {
+            request['orderIds'] = ids;
+            return await this.privateDeleteBulkOrders (this.extend (request, params));
+        }
+        if (clientOrderIds !== undefined) {
+            request['clientOrderIds'] = clientOrderIds;
```

Comment:
we're doing unnecessary work here because we're omitting `clientOrdersIds` from `params` and adding it to `request` as is. We don't need to do it because upon making the rest request, we extend `request` with `params`

```
this.extend (request, params)
```